modules/utility/preview/glow/glow.nix

```diff
@@ -0,0 +1,12 @@
+{lib, ...}: let
+  inherit (lib) mkEnableOption mkMappingOption;
+in {
+  options.vim.utility.preview = {
+    glow = {
+      enable = mkEnableOption "enable markdown preview in neovim with glow";
```

Comment:
`mkEnableOption` prefixes the description with "whether to enable" within its abstraction. In the documentation, this would look like like "whether to enable enable markdown preview in neovim with glow"